src/app/page.tsx

```diff
@@ -35,19 +35,37 @@ const Home = () => {
         <main className="flex min-h-0 grow flex-col items-center justify-between p-1">
           <SerialLoader>
             {isMac() && (
-              <p>
-                Looks like you are using a Mac. If you come across any issues,
-                please let us know{" "}
-                <a
-                  className="text-blue-400 underline"
-                  href="https://github.com/portapack-mayhem/MayhemHub/issues/43"
-                  target="_blank"
-                  rel="noopener noreferrer"
+              <div role="alert" className="alert alert-warning m-5 w-[70%]">
+                <svg
```

Comment:
Can we swap this for a FA icon?

Reply:
Ah, yep! I just copied the DaisyUI example 🫣